cdb2api/cdb2api.c

```diff
@@ -947,10 +952,12 @@ void cdb2_set_comdb2db_info(const char *cfg_info)
         CDB2DBCONFIG_BUF = NULL;
     }
     if (cfg_info == NULL) {
+        cdb2cfg_override = CDB2CFG_OVERRIDE_DEFAULT;
```

Comment:
Same as above, line should probably move into the reset_the_configuration() function.